public/js/components/FormFields/FormFieldTagPicker.js

```diff
@@ -0,0 +1,109 @@
+import React, {PropTypes} from 'react';
+import ShowErrors from '../Utilities/ShowErrors';
+import { errorPropType } from '../../constants/errorPropType';
+import {searchTags} from '../../services/capi';
+
+export default class FormFieldTagPicker extends React.Component {
+
+
+  static propTypes = {
+    fieldLabel: PropTypes.string,
+    fieldName: PropTypes.string,
+    fieldValue: PropTypes.string,
+    fieldPlaceholder: PropTypes.string,
+    fieldErrors: PropTypes.arrayOf(errorPropType),
+    formRowClass: PropTypes.string,
+    onUpdateField: PropTypes.func
+  };
+
+  state = {
+    suggestions: undefined,
+    searchText: ""
+  };
+
+  selectTag = (id) => {
+    this.props.onUpdateField(id);
+
+    this.setState({
+      suggestions: undefined,
+      searchText: ""
+    });
+  }
+
+  resetTag = () => {
+    this.props.onUpdateField(undefined);
```

Comment:
Can't this just be sent with no params? The param on the other side would be `undefined` anyway